elements/order-input.html

```diff
@@ -0,0 +1,97 @@
+<link rel="import" href="../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../bower_components/paper-button/paper-button.html">
+<link rel="import" href="../bower_components/paper-checkbox/paper-checkbox.html">
+<link rel="import" href="../bower_components/paper-input/paper-input.html">
+<dom-module id="order-input">
+    <template>
+        <style>
+            .hidden {
+                display: none;
+            }
+            paper-input {
+                /* display: inline-block; */
+                max-width: 100px;
+                --paper-input-container: { 
+                    padding: 0;
+                };
+            }
+            paper-checkbox {
+                margin-left: 10px;
+            }
+            paper-button {
+                margin-top: 10px;
+            }
+            #tipContainer {
+                display: flex;
+                align-items: flex-end;
+            }
+            span[slot="prefix"] {
+                padding-right: 2px;
+            }
+        </style>
+        <div>
+            <textarea id="textarea" rows="20" cols="60" placeholder="Paste order details here" autofocus></textarea>
+            <paper-input allowed-pattern="[0-9]" label="Taxes" id="tax">
```

Comment:
This will only allow whole numbers to be entered. We should switch this to `type="number"` along with the following ones